packages/frontend/components/atomic/BaseDropdown/BaseDropdown.tsx

```diff
@@ -37,7 +38,8 @@ const BaseDropdown = React.forwardRef<HTMLButtonElement, Props>(
           role={undefined}
           aria-label={label}
         >
-          {children}
+          {typeof children === "function" &&
+            children({ onClick: closePopover })}
```

Comment:
@jen-castiron Will you double check my types on this before I merge? Thanks!

Reply:
This check works, but if the children was not a function, it wasn't rendered. I added a small fix. 👍